clients/java/src/main/java/io/unitycatalog/client/auth/AuthConfigs.java

```diff
@@ -0,0 +1,21 @@
+package io.unitycatalog.client.auth;
+
+class AuthConfigs {
+  private AuthConfigs() {}
+
+  // Define the authentication type, where the type can be:
+  // 1. static: which uses the StaticTokenProvider to generate the static token.
+  // 2. oauth: which uses the OAuthTokenProvider to generate the oauth token.
+  // 3. fully qualified class name of the custom TokenProvider implementation.
+  static final String TYPE = "type";
+
+  // Configure keys for static token provider.
+  static final String STATIC_TYPE = "static";
```

Comment:
`STATIC_TYPE_VALUE`. to differentiate from `TYPE` which is the key.

Reply:
Done